share/completions/qdbus.fish

```diff
@@ -0,0 +1,36 @@
+function __fish_qdbus_complete
+    set -l argv (commandline --cut-at-cursor --tokenize)
+    argparse --name qdbus system 'bus=' literal help -- $argv
+    if set -q _flag_help
+        return
+    end
+    set -l qdbus_flags
+    if set -q _flag_system
```

Comment:
Argparse will store boolean flags as the flags themselves.

So $_flag_system will be set to `--system` or won't be set, so this can just be

```fish
set -l qdbus_flags $_flag_system
````